integer/integer.go

```diff
@@ -16,7 +16,29 @@ limitations under the License.
 
 package integer
 
+// number is a constraint that permits any floating-point type, any signed type, or any unsigned type
+type number interface {
+	~float32 | ~float64 | ~int | ~int8 | ~int16 | ~int32 | ~int64 | ~uint | ~uint8 | ~uint16 | ~uint32 | ~uint64 | ~uintptr
```

Comment:
Why do we handle float types in package `integer` ?

"numeric" ?  "mathutil"?  Something else?  Or don't handle float?

Reply:
Another option is to make a new package and just alias - `var IntMax = numeric.Max[int]` if you want to keep floats

Reply:
Nah, I think this is probably fine. If we needed it, I imagine someone would have created a float library by now.